statistics/table.go

```diff
@@ -239,3 +258,97 @@ func PseudoTable(ti *model.TableInfo) *Table {
 	}
 	return t
 }
+
+func getPseudoRowCountByIndexRanges(sc *variable.StatementContext, indexRanges []*types.IndexRange, inAndEQCnt int,
```

Comment:
It's not newly added, just moved from plan.
There are test cases in plan cover this.